torch/_inductor/fx_passes/group_batch_fusion.py

```diff
@@ -653,6 +739,9 @@ def group_batch_fusion_post_grad_passes(graph: torch.fx.Graph):
     if config.group_fusion and has_fbgemm:
         fusions += [GroupLinearFusion()]
 
+    if config.post_grad_batch_fusion:
+        fusions += [BatchLinearPostGradFusion()]
+
```

Comment:
I think we need a mechanism to only enable one batch linear fusion, either pre or post grad.

Reply:
Or probably we have a config can be used to specify a list of fusions and the order.